minimesos/src/main/groovy/com/containersol/minimesos/config/ClusterConfig.groovy

```diff
@@ -53,4 +53,14 @@ class ClusterConfig extends GroovyBlock {
         delegateTo(marathon, cl)
     }
 
+    void setLoggingLevel(String loggingLevel) {
+         if (!loggingLevel.equals("WARNING") && !loggingLevel.equals("INFO") && !loggingLevel.equals("ERROR")) {
+             throw new RuntimeException("Property 'loggingLevel' can only have the values INFO, WARNING or ERROR")
```

Comment:
WARNING, INFO and ERROR are the supported logging levels for Mesos http://mesos.apache.org/documentation/latest/configuration

Case insensitive, good point